README.md

```diff
@@ -1 +1,175 @@
-# capstone-2
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [toDoList] <a name="about-project"></a>
+
+**[PokeCard]** is a website that allows you to check pokemons and comment abou it!.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML, CSS and JavaScript
+- Git, Github and Visual Studio Code
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[show_pokemons]**
+- **[select_pokemon]**
+- **[consume_api]**
+- **[get_api]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install Github
+- Install VisualStudio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Choose a directory
+- Open a Terminal
+- write: git clone [here](git@github.com:Swarzstein/capstone-2.git)
+
+### Install
+
+Install this project with:
+
+- get into to directory "capstones-2"
+- write: npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+- In gitbash terminal
+- write: code .
+
+### Deployment
+
+You can deploy this project using:
+
+[Deploy](https://github.com/Swarzstein/capstone-2/dist/)
+
```

Comment:
[OPTIONAL]Please consider adding a functional live demo link. The current one is giving a 404 error